### PR TITLE
Make guild config util func

### LINKED ISSUE #37: Refactor: getGuildConfig()
- Currently, when we want to get the saved guildConfig for a guild, we use different ways and it's inefficient.
- Your task is to build a clean function that accepts Guild or guild ID as input and return `undefined` or the guild config object depending on whether it's present or not.
- Put the file in `src/utilities/` folder.
- While you're at it, replace all instances of guildConfigurations with your newly created function.

### PR DESCRIPTION
**Pull request by codevenkat**
This PR fixes #37 
* This also changes the inhibitor to take in strings instead of ints, so the config should probably change accordingly

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,7 +1,8 @@
 const { config } = require('dotenv');
 const { AkairoClient, CommandHandler, ListenerHandler, InhibitorHandler } = require('discord-akairo');
 const { join } = require('path');
-const configFile = require('../config.json')
+const configFile = require('../config.json');
```
Comment (ridays2001) [inline src/index.js:4]: Remove declaration for the useless file.

File: src/inhibitors/roleInhibitor.js [hunk 2]
```diff
@@ -1,5 +1,6 @@
 const { Inhibitor } = require('discord-akairo');
 const config = require('../../config.json');
```
Comment (ridays2001) [inline src/inhibitors/roleInhibitor.js:2]: Remove this useless import.

**Review by ridays2001 — CHANGES_REQUESTED** [review]
Remove useless imports from 2 places. Rest all looks fine.

**Comment by codevenkat** [comment]
isnt configFile.botOwner used directly now though?

**Comment by ridays2001** [comment]
Oh, right. Then remove only from the role inhibitor file. Sorry, didn't notice it

**Comment by codevenkat** [comment]
the inhibitor file uses `config.guildConfigurations.length === 0`, should that be replaced by `!getGuildConfig(message.guild.id)` maybe?

**Pull request merged**